ONSdigital/ras-secure-message
# Copy sent events into messages

**What is the context of this PR?**

https://trello.com/c/Fv73S3XV/223-migrate-events-data-into-securemessage-3

This is the second part of the migration away from the events table.  We need to have every message in the secure-message table to have the data stored in the events table, stored in itself.  This will copy every sent datetime from the events table into the sent_at field in secure_message.

**How to review**

- Create a number of conversations locally.  
- Open some of them to get read records.  
- Go into the database and delete any number of sent_at and read_at records in the secure-message table
- Run the script
- Check the database, all the sent_at and read_at records that were deleted should have reappeared and no others should have appeared.
- For piece of mind, maybe try it again, but delete a different number of records.
